Makefile

```diff
@@ -22,8 +22,8 @@ all: lz_header.bin
 # unreferenced section ahead of .text, in which case link.lds needs adjusting.
 lz_header.bin: lz_header Makefile
 	objcopy -O binary -S --pad-to 0x10000 $< $@
-	@od --format=x8 --skip-bytes=4 --read-bytes=16 $@ | \
-		grep "0000004 e91192048e26f178 02ccc4765bc82a83" > /dev/null || \
+	@od --format=x8 --skip-bytes=`hexdump -s2 -n2 -e '"%u"' $@` --read-bytes=16 $@ | \
+		grep "e91192048e26f178 02ccc4765bc82a83" > /dev/null || \
```

Comment:
This logic looks like it is going to be repeated often.  Seeing as cal_skinit_lz_sums is shell and can't reasonably be done in make, its probably best to add a util.sh which can be sourced by various scripts, containing a function to extract the measured length.

This od line can turn into sanity_check.sh

Reply:
Done